packages/forms/resources/views/components/field-wrapper/label.blade.php

```diff
@@ -20,7 +20,8 @@
         ])
     >
         {{-- Deliberately poor formatting to ensure that the asterisk sticks to the final word in the label. --}}
-        {{ $slot }}@if ($required)<span class="whitespace-nowrap">
+        {{ $slot }}@if ($required)
+            <span class="whitespace-nowrap">
```

Comment:
Could you try removing this wrapping `span` and moving `<sup>` onto the same line as `@if`? I recently noticed the asterisk doesn't stick to the final word anymore.